src/signMessage.c

```diff
@@ -82,7 +111,14 @@ void handleSignMessage(uint8_t p1, uint8_t p2, uint8_t *dataBuffer, uint16_t dat
 
         sendResponse(0, false);
     } else {
-        ux_flow_init(0, ux_display_message, NULL);
+        if (system_transfer_err == 0) {
+	    uint32_t sol = info.lamports / 1000000000;
+	    uint32_t lamports = info.lamports % 1000000000;
+            snprintf(messageTransfer, MESSAGE_TRANSFER_SIZE, "%u, %u", sol, lamports);
```

Comment:
This is super ugly. We need a library function for printing a decimal amount of SOL.